src/utils/mtev_confstr.c

```diff
@@ -169,6 +169,11 @@ mtev_confstr_parse_duration(const char *input, uint64_t *output,
     if(unit_str == input)
       return MTEV_CONFSTR_PARSE_ERR_FORMAT;
 
+    /* remove space(s) between number and unit string */
+    while(!isalpha(*unit_str)){
```

Comment:
Is this guaranteed to not loop off the end of the string?

Reply:
add `while(*unit_str && !isalpha(*unit_str))`